core/addRxns.m

```diff
@@ -214,7 +214,11 @@
     for i = 1:length(rxnsToAdd.mets)
         mets         = rxnsToAdd.mets{i};
         stoichCoeffs = rxnsToAdd.stoichCoeffs{i};
-        isrev        = rxnsToAdd.lb(i) < 0;
+        if isfield(rxnsToAdd,'ub')
+            isrev        = rxnsToAdd.lb(i) < 0 && rxnsToAdd.ub(i) > 0;
+        else
+            isrev        = rxnsToAdd.lb(i) < 0;
```

Comment:
a reaction can also be irreversible if lb 0 andt ub > 0. Then this line will result in a logical 0
```suggestion
            isrev        = rxnsToAdd.lb(i) == 0 ||  rxnsToAdd.ub(i) == 0;
```

Reply:
But note that line 220 is only run if no 'ub' is specified (`addRxns` allows for that). If no 'ub' is specified, `addRxns` will set ub to `defaultUB` or `1000`, so >0. If 'lb' is then <0, the reaction is reversible.

Reply:
Ok, I got it